projects/core/src/configurator/commons/facade/configurator-cart.service.ts

```diff
@@ -26,7 +26,7 @@ export class ConfiguratorCartService {
    * Checks if there are no updates for the active cart pending
```

Comment:
This description is not quite correct.
Change to: Checks if there are updates for the active cart pending.

Reply:
You are right. But I will delete those methods anyhow (subject of next PR on Larisas for checking), so doesn't make sense to change...

Reply:
Method will be deleted...